### ISSUE TITLE
Cannot set property 'muted' of null

### ISSUE THREAD

**Issue by peavers** [nkoehler/mat-video#45]
**Bug**

#### What is the current behavior?
When loading in a video with a simple configuration such as
```
        <mat-video
          src="{{ video.absolutePath | remoteVideo }}"
          [download]="true"
          [title]="video.name"
          [poster]="coverImage(video) | remoteImage"
        ></mat-video>
```
The following error is thrown in the console. 
```
core.js:5845 ERROR TypeError: Cannot set property 'muted' of null
    at MatVolumeControlComponent.set muted [as muted] (mat-video.js:1150)
    at setInputsForProperty (core.js:13504)
    at elementPropertyInternal (core.js:12319)
    at Module.ɵɵproperty (core.js:20842)
    at MatVideoComponent_div_8_Template (mat-video.js:96)
    at executeTemplate (core.js:11858)
    at refreshView (core.js:11705)
    at refreshDynamicEmbeddedViews (core.js:13070)
    at refreshView (core.js:11728)
    at refreshComponent (core.js:13145)
```
The video however does play and respond still as expected. 

#### What is the expected behavior?
No console error is thrown. 

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
```
"@angular/compiler": "~9.0.1",
"@angular/material": "^9.0.0",
"mat-video": "^2.7.2",
"typescript": "~3.7.5"
```

Chrome Version 80.0.3987.132
Mac OS 10.15.3

**Comment by then3rdman** [comment]
Also getting this exact behaviour

**Comment by AndriusdevLa** [comment]
Also getting cannot get muted of null

**Comment by chedydy** [comment]
same here

**Comment by born2net** [comment]
yep... same bug.. started appearing in last version... 
awesome project, please fix...
TX!!!!

**Comment by MichaelShi1979** [comment]
what's the status of this bug?

**Comment by peavers** [comment]
PR is open with a fix; just needs to be merged.

**Comment by born2net** [comment]
when can it be merged? tx!

**Comment by nkoehler** [comment]
Fixed in version 2.8.0.
